namespaces/default/snekbox-forms/deployment.yaml

```diff
@@ -12,6 +12,21 @@ spec:
       labels:
         app: snekbox-forms
     spec:
+      initContainers:
+        - name: init-service
+          image: busybox:latest
+          command: ["/bin/sh", "-c"]
+          args: 
+          - >
+            wget https://url.to.file.com/image.png -O /tmp/images/img1.png
```

Comment:
This is a placeholder until we decide on the actual image URL that will be used here.